js/app.js

```diff
@@ -1,2 +1,264 @@
 'use strict';
 
+var hoursOfOperation = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm', '8pm'];
+var dailyTotalSales = 0;
+var ulElement;
+var listElement;
+
+var firstAndPike = {
+  location: '1st and Pike',
+  minCustHour: 23,
+  maxCustHour: 65,
+  avgNumPer: 6.3,
+  custPerHour: [],
+  salesPerHour: [],
+  dailyTotal: 0,
+  storeMin: function(num) { //@param num minimum number of customers per hour
```

Comment:
This looks like documentation; I'm assuming you're doing this just for yourself, since we haven't talked about it yet? 

I'm a huge fan of documenting parameters! I'd suggest adding in the expected type of the variable to your documentation.